.Xamarin.Forms.Android.slnf

```diff
@@ -22,14 +19,12 @@
       "Xamarin.Forms.Maps\\Xamarin.Forms.Maps.csproj",
       "Xamarin.Forms.Material.Android\\Xamarin.Forms.Material.Android.csproj",
       "Xamarin.Forms.Pages.Azure\\Xamarin.Forms.Pages.Azure.csproj",
-      "Xamarin.Forms.Pages.UnitTests\\Xamarin.Forms.Pages.UnitTests.csproj",
       "Xamarin.Forms.Pages\\Xamarin.Forms.Pages.csproj",
       "Xamarin.Forms.Platform.Android.AppLinks\\Xamarin.Forms.Platform.Android.AppLinks.csproj",
       "Xamarin.Forms.Platform.Android.FormsViewGroup\\Xamarin.Forms.Platform.Android.FormsViewGroup.csproj",
+      "Xamarin.Forms.Platform.Android.UnitTests\\Xamarin.Forms.Platform.Android.UnitTests.csproj",
       "Xamarin.Forms.Platform.Android\\Xamarin.Forms.Platform.Android.csproj",
       "Xamarin.Forms.Platform\\Xamarin.Forms.Platform.csproj",
-      "Xamarin.Forms.Sandbox.Android\\Xamarin.Forms.Sandbox.Android.csproj",
-      "Xamarin.Forms.Sandbox\\Xamarin.Forms.Sandbox.csproj",
```

Comment:
Are these changes necessary?

Reply:
No, I'll revert that.

Reply:
Oh, wait - yes, they are necessary. The solution filter was still trying to include stuff we removed since it was created.